src/display/network.js

```diff
@@ -593,8 +569,6 @@ PDFNetworkStreamRangeRequestReader.prototype = {
   },
 };
 
-setPDFNetworkStreamClass(PDFNetworkStream);
```

Comment:
But in non-Node.js, the stream class is no longer set. That's not good.

It is safe to repeatedly call `setPDFNetworkStreamClass`, so restore this line.